x-pack/solutions/observability/packages/kbn-streams-schema/src/models/ingest/lifecycle/index.ts

```diff
@@ -46,12 +53,17 @@ export const ingestStreamLifecycleSchema: z.Schema<IngestStreamLifecycle> = z.un
   dslLifecycleSchema,
   ilmLifecycleSchema,
   inheritLifecycleSchema,
-  disabledLifecycleSchema,
 ]);
 
-export type InheritedIngestStreamLifecycle = IngestStreamLifecycle & { from: string };
+export const unwiredIngestStreamEffectiveLifecycleSchema: z.Schema<UnwiredIngestStreamEffectiveLifecycle> =
+  z.union([
+    dslLifecycleSchema,
+    ilmLifecycleSchema,
+    inheritLifecycleSchema,
+    disabledLifecycleSchema,
```

Comment:
I've removed `disabled` from the lifecycle input options. It is now only a read value for the `effective_lifecycle` of unwired streams